src/Symfony/Toggle.php

```diff
@@ -53,7 +54,7 @@ public function isActive(string $feature, array $context = []): bool
             self::$variables = [
                 'token' => $token,
                 'request' => $request,
-                'roles' => array_map(function (RoleInterface $role) {
+                'roles' => array_map(static function (Role $role) {
                     return $role->getRole();
                 }, $roles),
                 'trust_resolver' => $this->container->get('security.authentication.trust_resolver'),
```

Comment:
Regarding these, @pierredup  refer to comment https://github.com/SolidWorx/Toggler/pull/20/files#r537665040